### DISCUSSION TITLE
When starknet.js v4 could/should be deprecated?

### DISCUSSION THREAD

**Discussion by ivpavici** [seanjameshan/starknet.js#630] [Ideas]
As the title says... v5 will introduce a lot of improvements (albeit with breaking changes) together with Starknet protocol changes support.

This discussion is important because of https://github.com/starknet-io/get-starknet project as well.
One more thing to consider is that the Sequencer API will be turned off for users in the near future as well.

All comments and suggestions are welcome!

**Comment by PhilippeR26** [comment]
Regenesis could be a good milestone.

**Reply by ivpavici** [reply]
indeed, if the sequncer doesn't get deprecated before that!

**Reply by kravenh** [reply]
maybe...

**Comment by penovicp** [comment]
Something that might be worth mentioning here is that `v3` still gets a bunch of downloads per week on `npm`, `3.19.0` is at `3,548`, as a reference `4.22.0` is at `4,972`. Seems way too high to be the result of scraping.

**Comment by jeffkubatersalfons** [comment]
now work to slow starknet
